js/spec/index-spec.js

```diff
@@ -1,51 +1,49 @@
-const nsfw = require('../src/');
-const path = require('path');
-const fse = require('fs-extra');
+const assert = require('assert');
 const exec = require('executive');
+const fse = require('fs-extra');
```

Comment:
While you're cleaning up unused dependencies, would it make sense to move from `fs-extra` to `fs` here as well now that we're dropping Node 8 support? Or would that require more work than find/replace.

Reply:
No, `fs` doesn't support the operations I'm using in the specs. `fse.remove` handles recursive deletes on all platforms.